README.md

```diff
@@ -280,6 +280,17 @@ To configure OAuth2 in the processor you need to follow these steps:
       ```
       The obtained token will be valid for 1 week (then a new one must be requested from the API) and needs to be sent as an `Authorization` header in all the requests. If you don't send the token (when the `UseAuthToken` attribute is set in `True`), you will receive a `401 Unauthorized` response.
 
+### Video feeds the processor can consume from
```

Comment:
What about calling this section "Supported video formats"? or "Supported video feeds formats"?